hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OmUtils.java

```diff
@@ -199,6 +199,7 @@ public static boolean isReadOnly(
     case LookupFile:
     case ListStatus:
     case GetAcl:
+    case DBUpdates:
```

Comment:
I believe non-leader OMs will fail to service this request in HA cluster. Is that fine?

Reply:
Yes, for now this is OK. Handling of OM HA in recon will be done later.